docs/source/markdown/podman-attach.1.md

```diff
@@ -10,17 +10,16 @@ podman\-attach - Attach to a running container
 
 ## DESCRIPTION
 **podman attach** attaches to a running *container* using the *container's name* or *ID*, to either view its ongoing output or to control it interactively.\
-The *container* can detached from (and leave it running) using a configurable key sequence. The default sequence is `ctrl-p,ctrl-q`. Configure the keys sequence using the **--detach-keys** option, or specifying it in the `containers.conf` file: see **[containers.conf(5)](https://github.com/containers/common/blob/master/docs/containers.conf.5.md)** for more information.
+The *container* can detached from (and leave it running) using a configurable key sequence. The default sequence is `ctrl-p,ctrl-q`. Configure the keys sequence using the **--detach-keys** OPTION, or specifying it in the `containers.conf` file: see **[containers.conf(5)](https://github.com/containers/common/blob/master/docs/containers.conf.5.md)** for more information.
 
```

Comment:
We've another PR in flight with changes here too, I smell a merge conflict incoming.

Reply:
Yeah I saw that, Whatever is the outcome there, I'll change it in the other document in the future.